README.md

```diff
@@ -1 +1,18 @@
-# Welcome to OpenBC Website
\ No newline at end of file
+# Welcome to OpenBC
+
+## Project Set Up Instructions
+
+1. Clone the repo `openbc-web` using SSH
+2. `cd` into the repo
+3. Type `node -v` to ensure you have the latest version of Node.js installed, install Node.js or update your version if needed
```

Comment:
it would be nice to specify the version of node we expect to run this. latest can change over time. 
also take a look at nvm which allows you to pick your node version